### ISSUE TITLE
Pytorch-Tensor-Train-Network/tc/tc_math.py / Line 37  data = data.reshape(-1, a_raw_shape[1][-1], 1)

### ISSUE THREAD

**Issue by LouiValley** [uwjunqi/Tensor-Train-Neural-Network#2]
Excuse me , is there any mistake in your code ?

"data = data.reshape(-1, a_raw_shape[1][-1], 1)"

I think maybe it is 
"data = data.reshape(-1, a_raw_shape[1][**0**], 1)"

**Comment by huckiyang** [comment]
No, this is not a mistake. The value is inferred from the length of the array and remaining dimensions.
